docs/analysis/pysatModels.rst

```diff
@@ -0,0 +1,9 @@
+pysatModels
+-----------
+
+`pysatModels <https://github.com/pysat/pysatModels>`_ is a pysat interface for
+model analysis and model-data comparisons.  It allows model input to be supplied
+as a simple xarray Dataset, and compared to 'truth' data from a pysat object.
+See the
+`Utilities <https://pysatmodels.readthedocs.io/en/latest/utils.html>`_
```

Comment:
Here is what I see for the pysatModels Utilities link on RTD.

<img width="922" alt="Screen Shot 2021-02-04 at 4 17 16 PM" src="https://user-images.githubusercontent.com/3595641/106962297-8727e180-6704-11eb-8646-01fe27c0e3a9.png">

Reply:
Yeah, but that's a pysatModels issue.